core/src/main/groovy/org/grails/gorm/graphql/entity/dsl/helpers/Arguable.groovy

```diff
@@ -46,7 +46,7 @@ trait Arguable<T> extends ExecutesClosures {
      * @param closure To provide additional data about the argument
      * @return The operation in order to chain method calls
      */
-    T argument(String name, Class type, @DelegatesTo(value = SimpleArgument, strategy = Closure.DELEGATE_ONLY) Closure closure = null) {
+    T argument(String name, Class<?> type, @DelegatesTo(value = SimpleArgument, strategy = Closure.DELEGATE_ONLY) Closure closure = null) {
```

Comment:
Similar, in that when the implementing GORM entity used @GrailsCompileStatic, this would result in a static compilation error without the generic wildcard.